resultstore-search-api/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM python:3
+
+RUN pip install absl-py grpcio protobuf gcloud google-api-python-client google-api-python-client
+
+ENV APP_HOME /app
+WORKDIR $APP_HOME
+COPY . ./
```

Comment:
Make sure to check that this copy operation works correctly even if the Docker command is invoked outside of this directory.